log/log.go

```diff
@@ -15,12 +15,11 @@ var stdL atomic.Value
 func init() {
 	Replace(zap.NewExample())
 
-	fields := parseFields(os.Getenv("LOG_FIELDS"))
-	if fields == nil {
-		fields = parseFields(os.Getenv("LOG_TAGS"))
-	}
+	name := rucksack.Env("LOG_NAME", "APP_NAME")
```

Comment:
not a breaking change, but instead of a `MET_NAME` and ` LOG_NAME`, you can just specify `APP_NAME` env var which will be picked up by both